src/pages/login/PasswordReset.tsx

```diff
@@ -25,9 +29,9 @@ const PasswordReset: React.FC = () => {
       setMessage('Token is missing. Please try again.');
       return;
     }
-    AuthService.resetPassword(token, password).then(
```

Comment:
resetPassword vs set_new_password which one you think is camelCase naming convention?